src/data-access/properties.ts

```diff
@@ -0,0 +1,16 @@
+import { Client } from 'pg';
+import { Sequelize } from 'sequelize';
+
+const connectionString = 'postgres://bqgqthbd:9gOdo8uMOO6yQFufa-ypy2bFDW3SgeXz@dumbo.db.elephantsql.com:5432/bqgqthbd';
+
+export const sequelize = new Sequelize(connectionString);
```

Comment:
suggesting creating singleton of this and export `sequelize`  only, seems like you create multiple instances of same connection

Reply:
Fixed